### ISSUE TITLE
Panning in timeline is unintuitive

### ISSUE THREAD

**Issue by nicolae-stroncea** [ActivityWatch/activitywatch#446]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS version and name**:  Fedora 32, using Firefox
- **ActivityWatch version**:  0.92

## Describe the bug

For the longest time (1 year), I thought the panning in the timeline did not work, so I just stopped trying to use it. I only recently realized that to pan, you need to zoom in first.

## Possible solutions

1. Change the text *Drag to pan and scroll to zoom* to something more intuitive that explains you have to zoom in first
2. Zoom in a couple of percentage points by default, so user can pan from the beginning
3. Both

**Comment by ErikBjare** [comment]
This is because the timeline is locked to the interval of data that has been queried, we're really just using the interface provided by the timeline library we use. 

Ideally, I'd want a timeline-like visualization where it would automatically query events on zoom-out/panning, but that would likely be *a lot* of work for a relatively small UX gain, so not really a priority.

**Comment by nicolae-stroncea** [comment]
That would indeed be interesting to see but like you say probably would require significant effort.

My issue is a lot more simple: For me the phrasing ("Drag to pan and scroll to zoom.") implies that panning and zooming are independent, and that I should be able to drag freely. This is not the case, since I need to zoom to pan. Therefore changing the order of instructions / rephrasing them would go 95% to solving this:

**Comment by ErikBjare** [comment]
I'm open to suggestions for alternative phrasings (that aren't too wordy) :slightly_smiling_face:
